default/stringtables/en.txt

```diff
@@ -11743,31 +11744,43 @@ SR_WEAPON_0_1
 Flak Cannon
 
 SR_WEAPON_0_1_DESC
-The flak cannon does negligible damage to most ships, but is inexpensive and effective at destroying fighters.
+'''The flak cannon does negligible damage to most ships, but is inexpensive and effective at destroying fighters.
+
+Each flak cannon on a ship built by a species with the piloting trait will have an additional 1 shot per piloting level (-1 shot for Bad Pilots).'''
 
 SR_WEAPON_1_1
 Mass Driver
 
 SR_WEAPON_1_1_DESC
-The Mass Driver, a basic weapon.
+'''The Mass Driver, a basic weapon. Upgrading the weapon technology will increase the shot damage.
+
+Each mass driver on a ship built by a species with the piloting trait will do an additional 1 shot damage per piloting level (-1 for Bad Pilots).'''
```

Comment:
On second thought, I'll change "shot damage" to "damage" for the piloting effect. Since I state that it applies to each mass driver, we don't need to say "per shot". And since it is affecting "base" or "weapon" damage, don't want to get that confused with the "shot damage".